### PR TITLE
completed zh-cn translation

### PR DESCRIPTION
**Pull request by trifirew**
| File | Status |
| ---- | ------ |
| context.cson | 100% |
| menu_*.cson | 100% |
| settings.cson | 100% |

### CODE REVIEW

**Comment by trifirew** [comment]
@liuderchi When do you plan to release a new version? I can't wait to see my translation :)

**Comment by liuderchi** [comment]
@trifirew next release will include this PR and some fixes.
it's planned to release this weekend.

Thank you so much for translating the 'about' page in advanced (not yet posted on issue).
btw Happy Chinese New Year :tada:

File: README.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ One i18n package for multiple languages :earth_asia: :earth_americas: :earth_afr
...
-| Chinese(Simplified) |  `zh-cn` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3A%22help%20wanted%22%20label%3A%22i18n%20zh-cn%22) |
+| Arabic | `ar` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
+| Spanish | `es` | :white_check_mark: 100% |
+| German | `de` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
+| French | `fr` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
+| Hindi | `hi` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
+| Japanese | `ja` | :white_check_mark: 95% |
+| Korean | `ko` | :white_check_mark: 100% |
+| Dutch | `nl` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
+| Portuguese(Brazil) | `pr-br` | :exclamation: [I can translate](https://github.com/liuderchi/atom-i18n/blob/master/CONTRIBUTING.md) |
```
Comment (liuderchi) [inline README.md]: please edit only status of `zh-cn`

tip: if you want to update `README.md` by using only one extra commit, you can 
1. edit README then locally commit with `--amend` option
2. push it with `-f` option
3. check diff on PR page, if not satisfied restart step 1.

**Comment by trifirew** [comment]
@liuderchi README.md is fixed. 
I've just seen the new `about.cson` so I translated it.

Also,
_新春快樂，雞年大吉！_ 🎉

**Pull request merged**